### PR TITLE
Add color to release-sop

### PR DESCRIPTION
**Pull request by dskloetd**
# Motivation

It can be used to find specific values such as the commit hash or the RC URL in the output of the release SOP script.

# Changes

1. Output newly recorded values in green.
2. Output previously recorded values in magenta.
3. Update tests.

# Tests

updated

# Todos

- [ ] Add entry to changelog (if necessary).
not necessary

### CODE REVIEW

File: scripts/nns-dapp/release-sop [hunk 1]
```diff
@@ -150,7 +150,7 @@ checklist_add() {
   temp_file="$CHECKLIST_FILE.tmp"
   jq --arg name "$name" --arg value "$value" '. + [{name: $name, value: $value}]' "$CHECKLIST_FILE" >"$temp_file"
   mv "$temp_file" "$CHECKLIST_FILE"
-  echo "$name: $value"
+  echo -e "$name: \033[32m$value\033[0m"
```
Comment (mstrasinskis) [inline scripts/nns-dapp/release-sop:153]: Great improvement! Will it makes sense to replace `\033[35m`, `\033[35m` and the default colour sequence with a variables for readability (e.g. GREEN_COLOR, DEFAULT_COLOR)?

**Review by mstrasinskis — APPROVED** [review]
Great improvement, thank you! Already approved, just an optional suggestion.

File: scripts/nns-dapp/release-sop (hunk 1 above)
Comment (dskloetd) [reply scripts/nns-dapp/release-sop]: Good idea. Done.

**Review by mstrasinskis — APPROVED** [review]
Thanks! Re-approve.

**Pull request merged**